README.md

```diff
@@ -1,191 +1,122 @@
-  + [Evaluating Large Language Models](/EVALUATION_GUIDE.md)
-+ **Installing Lumigator**
-  + Building
-    + See below
-  + Using/Testing
-    + [Kubernetes Helm Charts](lumigator/infra/mzai/helm/lumigator/README.md)
-    + [Local install documentation](/.devcontainer/README.md)
-+ **Using Lumigator:**
-  + [Platform Examples](/notebooks/walkthrough.ipynb)
-  + [Lumigator API](/lumigator/README.md)
-  + Offline Evaluations with [lm-buddy](https://github.com/mozilla-ai/lm-buddy)
-+ **Extending Lumigator:**
-  + [Creating a new Lumigator endpoint](lumigator/python/mzai/backend/api/README.md)
+[The 2024 AI Index Report](https://aiindex.stanford.edu/wp-content/uploads/2024/05/HAI_AI-Index-Report-2024.pdf)
+highlighted that AI evaluation tools aren’t (yet) keeping up with the pace of development, making it
+harder for developers and businesses to make informed choices. Without a clear single method for
+comparing models, many teams end up using suboptimal solutions, or just choosing models based on
+hype, slowing down product progress and innovation.
 
+With Lumigator MVP, Mozilla.ai aims to make model selection transparent, efficient, and empowering.
+Lumigator provides a framework for comparing LLMs, using task-specific metrics to evaluate how well
+a model fits your project’s needs. With Lumigator, we want to ensure that you’re not just picking a
+model—you’re picking the right model for your use case.
 
-# Available Machine Learning Tasks
+## Get started
 
-## Summarization
+The simplest way to set up Lumigator is to deploy it locally using Docker Compose. To this end, you
+need to have the following prerequisites installed on your machine:
 
-### Models for Online Ground Truth Generation
+- A working installation of [Docker](https://docs.docker.com/engine/install/).
+    - On a Mac, you need Docker Desktop `4.3` or later and docker-compose `1.28` or later.
+    - On Linux, you need to follow the
+      [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/).
+- The system Python; no version manager, such as pyenv, should be active.
 
-| Model Type | Model                                        | via HuggingFace | via API |
-|------------|----------------------------------------------|-----------------|---------|
-| seq2seq    | facebook/bart-large-cnn                      |       X         |         |
-| causal     | gpt-4o-mini, gpt-4-turbo, gpt-3.5-turbo-0125 |                 |    X    |
-| causal     | open-mistral-7b                              |                 |    X    |
+You can run and develop Lumigator locally using Docker Compose. This creates three container
+services networked together to make up all the components of the Lumigator application:
 
+- `localstack`: Local storage for datasets that mimics S3-API compatible functionality.
+- `backend`: Lumigator’s FastAPI REST API.
+- `ray`: A Ray cluster for submitting several types of jobs.
 
-### Models for Offline Evaluation
```

Comment:
Where do users find currently supported models + tasks...? I didn't see it in a (quick) look through the other readmes + the doc - are we removing this info?

Reply:
@ividal Yes, my plan is to move this in the docs. But first, I have to find out if this information is up to date. For example, what does it mean "Models for Offline Evaluation"? Can I pick something that is not in this list? What if I want to evaluate a model for summarization that was published yesterday? Is this an exhaustive list and if yes why?